criu/cgroup.c

```diff
@@ -1064,37 +1069,41 @@ bool is_special_property(const char *prop)
 	return false;
 }
 
-static int userns_move(void *arg, int fd, pid_t pid)
+static int userns_move(void *args, int fd, pid_t pid)
```

Comment:
You can leave the name of the second argument empty since you don't use it

Reply:
For clarification, do you mean `int fd`?

Reply:
Sorry, I meant `pid` but it is true for `fd` too.

Reply:
Are you sure it is true for `fd` too? I considered using `err` in place of `fd, but it looks like a few lines down, `fd` is needed for closing the opened file + providing meaningful error messages. I'll make the `pid_t pid` change first and await additional clarification on the `int fd`. Thanks!

Reply:
Update: I remember why these three specific parameters are needed -- `userns_move` is passed as a "call" parameter for `userns_call`, which utilizes this line: `return call(arg, fd, getpid());`

Reply:
I wasn't asking you to remove those parameters, just to remove their names. That's something you can do in C++ but not in C, so I was wrong, sorry.

It doesn't matter now but I'm pretty sure it's true for fd too. Whatever's there it's clearly overridden in `err = fd = openat(cg, arg, O_WRONLY);`.